platformcomponents/desktop/attachment-file-error.json

```diff
@@ -0,0 +1,16 @@
+{
+  "attachment-file-error": {
+    "sent": {
+      "comment": "Applied to sent & blocked file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5533%3A11688",
+      "background": "@theme-background-primary-disabled",
+      "border": "@theme-outline-secondary-normal",
+      "iconForeground": "@theme-common-text-white",
+      "iconBackground": {
+        "default": "@theme-background-accent-normal"
+      },
+      "textFileName": "@theme-text-primary-disabled",
+      "textFileSize": "@theme-text-primary-disabled"
```

Comment:
like in the other part, better to group them:
```
"label":{
     "primary": "@theme-text-primary-disabled",
     "secondary": "@theme-text-secondary-disabled"
}
```

Reply:
Pushed commit
`Grouped texts under label key in attachment-file-error`
Changing this conversation to Resolved.